arbos/programs/api.go

```diff
@@ -267,6 +271,20 @@ func newApiClosures(
 		}
 		return common.Hash{}, cost
 	}
+	accountCode := func(address common.Address) ([]byte, uint64) {
+		cost := vm.WasmAccountTouchCost(evm.StateDB, address)
+		if !evm.StateDB.Empty(address) {
+			return evm.StateDB.GetCode(address), cost
+		}
+		return []byte{}, cost
```

Comment:
We shouldn't charge a constant amount of gas for a linear amount of work. 

Also no need to check if the account is empty (see `vm.opExtCodeCopy` in `instructions.go`)